core/common/src/main/java/alluxio/exception/status/AlluxioStatusException.java

```diff
@@ -111,6 +111,13 @@ public StatusException toGrpcStatusException() {
     return mStatus.asException();
   }
 
+  /**
+   * @return a {@link StatusRuntimeException} for this exception
+   */
+  public StatusRuntimeException toGrpcStatusRuntimeException() {
+    return mStatus.asRuntimeException();
```

Comment:
this is a good addition, but is unrelated to this PR, please open another PR.

Reply:
Done